frontend/public/locales/en/cluster.json

```diff
@@ -582,7 +582,7 @@
     "template.modal.linkProvider.title": "Link an Ansible connection",
     "template.modal.linkProvider.submit": "Link",
     "template.modal.linkProvider.submitting": "Linking",
-    "template.modal.noCredentials": "You don't have any Ansible credentials. Visit the <bold>Credentials</bold> tab and select <bold>Add credentials</bold> to create an Ansible credential.",
+    "template.modal.noCredentials": "You don't have any Ansible credentials. Visit the <bold>Credentials</bold> tab and select <bold>Add credential</bold> to create an Ansible credential.",
```

Comment:
Thanks @mark-nc . You are remembering correctly. The general rule at IBM was to not use them. We still try to avoid them in the formal docs for translation and clarity. Before we left there, and it continues here at Red Hat, there is more leeway toward being a little more conversational in some of the content, like the UI, blogs, and so on. For the empty states, that is the terminology that we are using to make it a little more conversational. We are trying to be consistent across the different areas of the product, so this should be ok.